x-pack/plugin/esql/qa/testFixtures/src/main/resources/meta.csv-spec

```diff
@@ -259,6 +261,7 @@ ends_with     |Returns a boolean that indicates whether a keyword string ends wi
 floor         |Round a number down to the nearest integer.
 from_base64   |Decode a base64 string.
 greatest      |Returns the maximum value from multiple columns. This is similar to <<esql-mv_max>> except it is intended to run on multiple columns at once.
+ip_prefix     |Returns the original IP with all but the first N bits set to zero.
```

Comment:
This file used to be so much worse to edit. It's still a pain.

Reply:
I was planning to make tomorrow a quick thread about this. IMO, we should remove those tests. As commented in the other comment, making functions is a chore, and this part was the worst imo. And adds vert little value